### PR TITLE
ID changed to guid and added locale

### PR DESCRIPTION
**Pull request by wachterjohannes**
Changed table structure to use guid instead of integer ids and added locale to the task.

Basic PR: https://github.com/sulu/sulu/pull/3037

### CODE REVIEW

File: src/Sulu/Bundle/AutomationBundle/Controller/TaskController.php [hunk 1]
```diff
@@ -61,10 +61,29 @@ public function cgetAction(Request $request)
             $listBuilder->where($fieldDescriptors['entityId'], $entityId);
         }
 
+        $ids = null;
+        if ($request->get('ids') !== null) {
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/AutomationBundle/Controller/TaskController.php:68]: this was a bug in the prior pull-requests.

**Comment by danrot** [comment]
The locale is correctly written to the database, but in the tab all tasks are shown, not only the ones for the selected language.

**Comment by danrot** [comment]
Apart from that, I also found out that the overlay immediately closes, without showing a loader. If the request fails it does so silently.

But that's probably out of scope of this PR.

**Comment by wachterjohannes** [comment]
@danrot fixed locale bug. for the loader i have added a todo entry in the main PR

**Comment by danrot** [comment]
ACK

**Pull request merged**